server/app/database/data/weapons.json

```diff
@@ -24149,40 +24171,65 @@
     "level": 108,
     "setID": "68",
     "stats": [
+      {
+        "stat": "Vitality",
+        "minStat": 81,
+        "maxStat": 150
+      },
+      {
+        "stat": "Strength",
+        "minStat": 31,
+        "maxStat": 50
+      },
       {
         "stat": "Agility",
-        "minStat": 26,
-        "maxStat": 40
+        "minStat": 31,
+        "maxStat": 50
       },
       {
-        "stat": "Vitality",
-        "minStat": 41,
-        "maxStat": 60
+        "stat": "Wisdom",
+        "minStat": 16,
+        "maxStat": 20
       },
       {
         "stat": "Critical",
-        "minStat": 2,
-        "maxStat": 3
+        "minStat": 1,
+        "maxStat": 2
       },
       {
-        "stat": "Damage",
-        "minStat": 3,
-        "maxStat": 5
+        "stat": "Neutral Damage",
+        "minStat": 4,
+        "maxStat": 6
       },
       {
-        "stat": "Trap Damage",
-        "minStat": 6,
-        "maxStat": 10
+        "stat": "Earth Damage",
+        "minStat": 4,
+        "maxStat": 6
       },
       {
-        "stat": "Power (traps)",
-        "minStat": 21,
-        "maxStat": 30
+        "stat": "Air Damage",
+        "minStat": 4,
+        "maxStat": 6
+      },
+      {
+        "stat": "Prospecting",
+        "minStat": 4,
+        "maxStat": 6
       },
       {
         "stat": "Initiative",
-        "minStat": null,
-        "maxStat": 100
+        "minStat": 101,
+        "maxStat": 150
+      },
+      {
+        "stat": "AP Parry",
+        "minStat": 3,
+        "maxStat": 5
+      },
+      {
+        "stat": "MP Parry",
+        "minStat": 3,
+        "maxStat": 5
       }
     ],
     "weaponStats": {
```

Comment:
weapon damage has been modified for the daggers. 13-22 neutral and 4-6 air steal